docs/document/docs/guides/in-office-day-list-customization.md

```diff
@@ -239,14 +243,14 @@ Now, let's enable the screen reader and try moving the focus to the button to se
   });
 ```
 
-#### The display on UI
 ![render](/img/tooltip_describeChild_false.gif)
```

Comment:
This image has poor resolution compared with the second gif image.
Could you please improve that? I can see the vertical and horizontal light lines in the image.

Reply:
Please check this again!